webapp/channels/src/components/markdown/markdown.tsx

```diff
@@ -124,7 +123,6 @@ function Markdown({
     }
 
     const inputOptions = Object.assign({
-        autolinkedUrlSchemes,
```

Comment:
To make it so that the new URL filter is applied evenly, I needed to make sure that all of the Markdown code everywhere passed this, even if it might not have before. Because of that, this config setting is now accessed in `Markdown.format` to save us having to pass it in everywhere